pom.xml

```diff
@@ -201,6 +212,7 @@
           <configuration>
             <useSystemClassLoader>false</useSystemClassLoader>
             <groups>unit</groups>
+            <argLine>@{argLine} -javaagent:${settings.localRepository}/org/mockito/mockito-core/${mockito.version}/mockito-core-${mockito.version}.jar</argLine>
```

Comment:
I noticed you've added the mockito-core java agent to the maven-surefire-plugin and maven-failsafe-plugin configurations. While this works for enabling mocking of final
  classes, a more modern and common approach is to use the mockito-inline dependency. This would make the pom.xml cleaner by removing the need for the <argLine>
  configuration. Have you considered this alternative?